cloud/deploy/deploy.go

```diff
@@ -167,6 +176,52 @@ func Deploy(path, runtimeID, wsID, pytest, envFile, imageName, deploymentName st
 		return nil
 	}
 
+	if dagDeploy != "" {
+		currentDeployment, err := deployment.GetDeployment(wsID, runtimeID, deploymentName, client)
+		if err != nil {
+			return err
+		}
+
+		runtimeID = currentDeployment.ID
+		scheduler := astro.Scheduler{}
+		scheduler.AU = currentDeployment.DeploymentSpec.Scheduler.AU
+		scheduler.Replicas = currentDeployment.DeploymentSpec.Scheduler.Replicas
+		spec := astro.DeploymentCreateSpec{
+			Executor:  "CeleryExecutor",
```

Comment:
nitpick: might want to make this a constant, and is this okay to hardcode, I don't have much idea about how Astro works but can deployment have any other executor type?

Reply:
Not for now. By default all deployments use Celery Executors